### PR TITLE
refactor: PSR4 namespaces + cleanup of namespaces

### PR DESCRIPTION
**Pull request by odinuv**
v priprave na https://github.com/keboola/input-mapping/pull/78 vycisteni namespace
https://keboola.atlassian.net/browse/PS-1627

### CODE REVIEW

**Review by MiroCillik — APPROVED** [review]
Preletel som to, myslim, ze ok.

File: src/Table/Strategy/Synapse.php [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-namespace Keboola\InputMapping\Reader\Strategy;
+namespace Keboola\InputMapping\Table\Strategy;
 
-use Keboola\InputMapping\Reader\Options\InputTableOptions;
-use Keboola\InputMapping\Reader\WorkspaceProviderInterface;
+use Keboola\InputMapping\Table\Options\InputTableOptions;
+use Keboola\InputMapping\WorkspaceProviderInterface;
 
-class SynapseStrategy extends AbstractStrategy
+class Synapse extends AbstractStrategy
```
Comment (pivnicek) [inline src/Table/Strategy/Synapse.php:8]: and here not:).

**Review by pivnicek — APPROVED** [review]
the new namespaces are great

I left there a couple of irrelevant comments :-)

File: src/Table/Strategy/Redshift.php [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-use Keboola\InputMapping\Reader\LoadTypeDecider;
-use Keboola\InputMapping\Reader\Options\InputTableOptions;
-use Keboola\InputMapping\Reader\WorkspaceProviderInterface;
+use Keboola\InputMapping\Helper\LoadTypeDecider;
+use Keboola\InputMapping\Table\Options\InputTableOptions;
+use Keboola\InputMapping\WorkspaceProviderInterface;
 
-class RedshiftStrategy extends SnowflakeStrategy
+class Redshift extends Snowflake
```
[the remark this answers is not in the document]
Comment (odinuv) [reply src/Table/Strategy/Redshift.php:9]: yes it is - it's like that from the beginning https://github.com/keboola/input-mapping/commit/b34e327adf97348246bf79f554b3f149e0dbc1f7#diff-df65054734f57c40533fd0f340ceee455784b698c3f293a1c66ec56369aea983R9 Yep, should be cleaned up, but someothertime

File: src/Table/Strategy/Synapse.php (hunk 1 above)
Comment (odinuv) [reply src/Table/Strategy/Synapse.php:8]: yes, later and better way

File: composer.json [hunk 3]
```diff
@@ -16,20 +16,23 @@
...
         "keboola/storage-api-client": "^11.2",
         "keboola/storage-api-php-client-branch-wrapper": "^1.0.0",
-        "guzzlehttp/guzzle": "^6.5.5"
+        "ext-json": "*"
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "^3.5.8",
         "keboola/php-temp": "^1.0",
-        "phpunit/phpunit": "^5.7.27",
-        "codeclimate/php-test-reporter": "^0.4.4"
```
[the remark this answers is not in the document]
Comment (odinuv) [reply composer.json:25]: this kind of test reporter is long deprecated, the new one has to be built into pipeline and i haven't got to do that yet

**Pull request merged**